src/test/rustdoc-json/reexport_method_from_private_module.rs

```diff
@@ -0,0 +1,24 @@
+// Regression test for <https://github.com/rust-lang/rust/issues/102583>.
```

Comment:
Nit: This should be in `src/test/rustdoc-json/reexport.rs`

Reply:
:+1: